pkg/aws/aws.go

```diff
@@ -24,19 +24,27 @@ func AnalyzeWaste() {
 	}
 
 	var unusedResources []AWSResource
+
+	// Run all the checks
 	unusedAddresses, err := client.GetUnusedElasticIPAddresses(context.TODO())
 	if err == nil {
 		for _, address := range unusedAddresses {
 			unusedResources = append(unusedResources, address)
 		}
 	}
-
 	unusedGateways, err := client.GetUnusedNATGateways(context.TODO())
-	if err != nil {
+	if err == nil {
 		for _, gateway := range unusedGateways {
 			unusedResources = append(unusedResources, gateway)
 		}
```

Comment:
```suggestion
		unusedResources = append(unusedResources, gateway...)
```
I'm pretty sure we can just unroll the slice like this?

Reply:
nope :(